src/main/java/com/dougnoel/sentinel/steps/CsvSteps.java

```diff
@@ -0,0 +1,142 @@
+    }
+
+    /**
+     * Sets the given file to be the current file-under-test with the given number of header rows.
+     * @param fileLocation String either the location of the file, given by a path, or the name of a testdata object in the current page object.
+     * @param numberOfHeaderRows int the number of header rows in the CSV file to test.
+     * @throws FileNotFoundException In the case that the file is not found in the location specified.
+     */
+    @When("^I open (.*) as a (?:CSV|csv) file with (\\d+) header rows?$")
+    public static void openSpecificFileAsCsv(String fileLocation, int numberOfHeaderRows) throws FileNotFoundException {
+        String filePath;
+        try{
+            filePath = Configuration.getTestdataValue(fileLocation.trim(), "fileLocation");
+        }
+        catch(FileException fe){
+            filePath = fileLocation;
+        }
+
+        FileManager.setCurrentTestFile(new CsvFile(Path.of(filePath), numberOfHeaderRows));
+    }
+
+    /**
+     * Edits the current CSV file, setting every cell in the given column to the given value.
+     * @param column String column the name of the column, or an ordinal (1st, 2nd, 25th, etc.).
+     * @param desiredValue String the value to set each cell to.
+     */
+    @When("^I set all values in the (.*) column to (.*) in the (?:CSV|csv) file$")
+    public static void setAllColumnValuesInCsvFile(String column, String desiredValue){
+        CsvFile file = (CsvFile) FileManager.getCurrentTestFile();
+
+        String firstColumnCharacter = column.substring(0, 1);
+        if(StringUtils.isNumeric(firstColumnCharacter)){
+            file.setAllCellsInColumn(SentinelStringUtils.parseLeadingInt(column), desiredValue);
+        }
+        else{
+            file.setAllCellsInColumn(column, desiredValue);
+        }
+    }
+
+    /**
+     * Verifies the current CSV file has or does not have the given text in the given column and given row.
+     * @param assertion String if null is passed, looks for match(es), if any strong value is passed, looks for the value to not exist.
+     * @param matchType String whether we are doing an exact match or a partial match.
+     * @param textToMatch String the text to look for in the cell.
+     * @param column String column the name of the column, or an ordinal (1st, 2nd, 25th, etc.).
+     * @param rowNum String the row number. Can be "la" to specify the last row, or an integer.
+     */
+    @Then("^I verify the (?:CSV|csv) file( do(?:es)? not)? (has|have|contains?) the value (.*) in the (.*) column and the (la|\\d+)(?:st|nd|rd|th) row$")
+    public static void verifyCsvCellHasValue(String assertion, String matchType, String textToMatch, String column, String rowNum) {
+        CsvFile file = (CsvFile) FileManager.getCurrentTestFile();
+        boolean negate = !StringUtils.isEmpty(assertion);
+        int rowIndex = rowNum.equals("la") ? file.getNumberOfRows() : Integer.parseInt(rowNum);
```

Comment:
I could be blind but where do you handle the "la" case? I couldn't find it.

Reply:
It's in line 82: `rowNum.equals("la")`